CONTRIBUTING.md

```diff
@@ -9,3 +9,7 @@ By contributing to this project you agree to the Developer Certificate of
 Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
+
+## Contributing a new design document
+
+Please follow design documentation template design/_template.md.
```

Comment:
The underscore in the file name probably means we need to escape the whole name with back-quotes to make it render properly.

Reply:
Actually now that I can see it in the GitHub UI, it looks fine.